modmesh/system.py

```diff
@@ -111,4 +114,14 @@ def enter_main(argv):
         sys.stderr.write('mode "{}" is not supported'.format(args.mode))
     return ret
 
+
+def exec_code(code):
+    try:
+        apputil.run_code(code)
+    except Exception as e:
```

Comment:
this doesn't work well I think.
for example, I enter `print(a)` (with undefined `a`) in the console, it will just crash.